jenkins-run.py

```diff
@@ -210,7 +210,7 @@ def get_extends_recursive(self, configfiles, file_or_url):
     def _download_file(self, url):
         """ Download file from *url*, store it in a tempfile and return its path
         """
-        print '  download', url
+        print(f"  download ${url}")
```

Comment:
All the `f""` strings will break the python 2 compatibilty, maybe use `''.format`?